server/back-end/routes/miah.routes.js

```diff
@@ -0,0 +1,9 @@
+module.exports = app => {
+    const User = require("../controllers/miah.controllers.js");
+
+    // Create a new user
+    app.post("/users", User.create);
```

Comment:
As a general rule (this is more of a preference thing), I think routes on the back-end should have a "/api" in front of the requests. This is to prevent confusion between the routes of the front-end and back-end. 
E.g. https://localhost:8000:/users